ownCloud/Settings/MoreSettingsSection.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  MoreSettingsSection.swift
+//  ownCloud
+//
+//  Created by Pablo Carrascal on 03/05/2018.
+//  Copyright © 2018 ownCloud GmbH. All rights reserved.
+//
+
+/*
+ * Copyright (C) 2018, ownCloud GmbH.
+ *
+ * This code is covered by the GNU Public License Version 3.
+ *
+ * For distribution utilizing Apple mechanisms please see https://owncloud.org/contribute/iOS-license-exception/
+ * You should have received a copy of this license along with this program. If not, see <http://www.gnu.org/licenses/gpl-3.0.en.html>.
+ *
+ */
+
+import UIKit
+import WebKit
+import MessageUI
+
+// MARK: - Section identifier
+private let MoreSectionIdentifier: String = "settings-more-section"
```

Comment:
Please consider just using it in place if an identifier is only used once and private to a class.